eta/core/video.py

```diff
@@ -1599,19 +1598,18 @@ def duration(self):
         logger.warning("Unable to determine duration; returning -1")
         return -1
 
-    def get_raw_value(self, key):
-        '''Gets a value from the raw stream info dictionary.
-
-        Args:
-            key: the key to lookup in the stream info dictionary
-
-        Returns:
-            the value for the given key
-
-        Raises:
-            KeyError: if the key was not found in the stream info dictionary
+    @property
+    def size_bytes(self):
```

Comment:
@tylerganter `size_bytes` was not exposed as a property of `VideoStreamInfo`, but it should have been, so I added it here